pipelines/build/prTester/README.md

```diff
@@ -106,10 +86,13 @@ If you are on either list, the PR tester will run against your PR whenever you c
   - One that is at the front of the queue and currently being tested:
   ![Image of building tester](./images/pr_tester_building.png)
 
+- When the tester begins, it will generate several downstream jobs. Each of which will have their own job status:
+  ![Image of downstream Job](./images/pr_tester_downstream.png)
+
 - When the tester is done, it will return a response comment to the PR with feedback on the testing similar to the following:
 ![Image of test result](./images/pr_tester_result.png)
 
-- The message will vary depending on the result of the test. Please remember however, that failed tests may be due to existing problems in the nightly builds, not your code. If you're unsure if the tests failed because of your changes or not, check our [issue board](https://github.com/AdoptOpenJDK/openjdk-build/issues) and our [triage doc](https://docs.google.com/document/d/1vcZgHJeR8rW8U8OD23Uob7A1dbLrtkURZUkinUp7f_w/edit?usp=sharing) for the existing error. If your job was aborted, check the log to see who aborted it.
+- The message will vary depending on the result of the test. Please remember however, that failed tests may be due to existing problems in the nightly builds, not your code. If you're unsure if the tests failed because of your changes or not, check our [issue board](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/issues) and our [triage doc](https://docs.google.com/document/d/1vcZgHJeR8rW8U8OD23Uob7A1dbLrtkURZUkinUp7f_w/edit?usp=sharing) for the existing error. If your job was aborted, check the log to see who aborted it.
```

Comment:
It might be a good idea to include both the link to openjdk-build and ci-jenkins-pipelines, rather than replacing one with the other, as issues can come from both. Thoughts?

Reply:
PR tester issues are likely to be only related to ci-jenkins-pipeline code IMO. All openjdk-build changes are tested by a separate CI system before being merged so should be stable

Reply:
Okie dokie.